frontend/src/index.html

```diff
@@ -79,6 +79,98 @@
         </div>
       </div>
     </nav>
+
+    <!-- player UI design -->
+    <div class="my-player text-test fixed z-10 bottom-0 w-full bg-white border border-solid border-gray-300 rounded-t-md shadow-upward">
+      <div class="back absolute z-0 w-full h-full"></div>
+      <div class="head relative">
+        <div class="front relative pb-0 pt-2">
+          <div class="info relative px-4 w-full">
+            <h3 class="text-md break-words font-semibold">Q&A - The Love Between Krsna and Vrajavasis, How to Attain Perfection, etc.</h3>
```

Comment:
I think we should not use `h3` here, because the audio player will not be a part of the document outline. Just use `div`.

Reply:
Not fixed yet.